core/src/test/java/org/apache/iceberg/TestCatalogUtil.java

```diff
@@ -142,7 +141,7 @@ public void loadCustomFileIO_hadoopConfigConstructor() {
     FileIO fileIO =
         CatalogUtil.loadFileIO(HadoopFileIO.class.getName(), Maps.newHashMap(), configuration);
     Assertions.assertThat(fileIO).isInstanceOf(HadoopFileIO.class);
-    Assert.assertEquals("val", ((HadoopFileIO) fileIO).conf().get("key"));
+    Assertions.assertThat(((HadoopFileIO) fileIO).conf().get("key")).isEqualTo("val");
```

Comment:
```suggestion
    Assertions.assertThat(((HadoopFileIO) fileIO).conf()).containsEntry("key", "val");
```

Reply:
I tried your suggestion, but I get a syntax error.  I reverted back to `Assertions.assertThat(((HadoopFileIO) fileIO).conf().get("key")).isEqualTo("val");.`